src/api/app/helpers/meta_tag_helper.rb

```diff
@@ -0,0 +1,13 @@
+module MetaTagHelper
+  def meta_title
+    content_for?(:meta_title) ? content_for(:meta_title) : 'Open Build Service'
+  end
+
+  def meta_image
+    content_for?(:meta_image) ? content_for(:meta_image) : image_url('obs-logo_small.svg')
```

Comment:
SVG images will not be supported as display by many providers, due to possibility of executing code within SVG (via embedded js), use PNG instead (you might need to render SVG to PNG in assets dir).

Reply:
Good catch. I made a PNG and used that instead.